src/utils.ts

```diff
@@ -285,3 +285,9 @@ export function replaceKeys(obj: Record<string, any>): Record<string, any> {
     return obj;
   }
 }
+
+export function getEntries<Item>(obj: ArrayLike<Item> | Record<string, Item>, options: GlobalContext) {
+  const entries = Object.entries(obj);
+  if (options.alphabetize) entries.sort(([a], [b]) => a.localeCompare(b, "en"));
```

Comment:
```suggestion
  if (options.alphabetize) entries.sort(([a], [b]) => a.localeCompare(b, "en", { numeric: true }));
```

Small little creature comfort: add [numeric sort option](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#numeric_sorting) (sorts `1`, `2`, `10` rather than the default `1`, `10`, `2`)